src/celeritas/ext/GeantImporter.cc

```diff
@@ -1272,26 +1270,22 @@ import_volumes(G4VPhysicalVolume const& world, bool unique_volumes)
             {
                 volume.phys_material_id = cuts->GetIndex();
             }
-            volume.name = lv.GetName();
+            // TODO: when changing to celeritas::inp, just make this a label
+            // instead of converting to and from a std::string
+            volume.name = to_string(labels[lv.GetInstanceID()]);
```

Comment:
Should we just move from `volume.name` to `volume.label` at this point?

Reply:
Correct, we'd end up adding `celeritas::Label` to the root dictionary and updating things accordingly. This ties up with a discussion a while ago about the input structure and less indirections, such as directly storing opaque ids instead of ints, for example. I also see this quickly becoming a rabbit hole, so happy to defer this to a future discussion and PR whose focus is to improve our root I/O library.

Reply:
Yeah let's postpone that until the refactor.